data-raw/Age-Pension-assets-test-1997-present.tsv

```diff
@@ -113,7 +117,7 @@ TRUE	TRUE	FALSE	2015-07-01	440500
 TRUE	TRUE	FALSE	2016-07-01	448000
 TRUE	TRUE	FALSE	2017-01-01	575000
 TRUE	TRUE	FALSE	2017-07-01	583500
-TRUE	TRUE	FALSE	2018-07-01	594500
+TRUE	TRUE	FALSE	2019-07-01	605000
```

Comment:
Is this mean to be an insertion, rather than a replacement?

Reply:
oh god, yes it is. I'll fix.

Reply:
I suspect it's a problem with the logic with `Age_pension_assets_test_by_year <- ` where it uses the original data regardless of its age. The original logic was due to the the `      htmltab::htmltab("http://guides.dss.gov.au/guide-social-security-law/4/10/3", which = 10)` being too flaky to run every time.

Reply:
fixed now